net/mikrotik-btest/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2016 Roger Pueyo Centelles <roger.pueyo@guifi.net>
+#
+# This is free software, licensed under the GNU General Public License v3
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mikrotik_btest
+PKG_VERSION:=0.5.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/kadosch/mikrotik_btest/archive/$(PKG_VERSION)
```

Comment:
`PKG_SOURCE` will eventually clash with other package,
best is to use `PKG_SOURCE_PROTO:=git`

Reply:
Thanks, @champtar . Requested changes have been made.